Binary files kylin-recorder-1.3.0/.git/index and kylin-recorder-1.3.0/.git/index differ
diff -u -r kylin-recorder-1.3.0/src/menumodule.cpp kylin-recorder-1.3.0/src/menumodule.cpp
--- kylin-recorder-1.3.0/src/menumodule.cpp	2021-12-27 11:36:58.000000000 +0800
+++ kylin-recorder-1.3.0/src/menumodule.cpp	2023-06-17 20:50:35.000000000 +0800
@@ -1,7 +1,7 @@
 #include "menumodule.h"
 #include "mainwindow.h"
 #include "xatom-helper.h"
-menuModule::menuModule(QWidget *parent = nullptr) : QWidget(parent)
+menuModule::menuModule(QWidget *parent) : QWidget(parent)
 {
     init();
 }
